src/TimeService.Business/Commands/LeaveTime/FindLeaveTimesCommand.cs

```diff
@@ -31,25 +36,38 @@ public class FindLeaveTimesCommand : IFindLeaveTimesCommand
     private readonly IHttpContextAccessor _httpContextAccessor;
     private readonly IRequestClient<IGetUsersDataRequest> _rcGetUsers;
     private readonly ILogger<FindLeaveTimesCommand> _logger;
+    private readonly IConnectionMultiplexer _cache;
 
-    private List<UserInfo> GetUsers(List<Guid> userIds, List<string> errors)
+    private async Task<List<UserData>> GetUsersData(List<Guid> userIds, List<string> errors)
```

Comment:
would you like to check userIds != null?

Reply:
fixed